src/core/selectors.ts

```diff
@@ -133,6 +138,12 @@ namespace Selectors {
 
   export const recommendationsProducts = (state: Store.State) =>
     state.data.present.recommendations.products;
+
+  export const uiTagName = (state: Store.State, tagName: string) =>
```

Comment:
I think a more appropriate name for this would be `tagStates()` or `uiTagStates()`, because that's the name of what you're actually selecting